contributing.md

```diff
@@ -0,0 +1,38 @@
+# કંિટ્રબ્યુટિંગ
+
+નમસ્તે! અમે રોમાંચિત છીએ કે તમે આ પ઼ોજૅકટ માં યોગદાન કરવા ઈચ્છો છો. પ઼ોજેકટ સારો બનાવી રાખવા માટે તમારી મદદ જરુરી છે.
+
+આ ડોક્યુમૅન્ટ તમારી આ પ઼ોજૅકટ માં યોગદાન કરવા મદદ કરશે.
+
+# પૂલ રિક્વેસ્ટ કેવી રીતે કરવી
+
+- આ [issue](https://github.com/gatsbyjs/gatsby-gu/issues/1) પર જાવ અને જેમને તમે ગુજરાતી માં ટ્રાંસલેટ કરવા ઈચ્છો છો તેનુ પેજ઼ નામ કમેંટ કરો.
+- રિપાૅઝીટરી ને fork કરો.
+- Fork કરેલી રિપાૅઝીટરી ને ક્લોન કરો. (અથવા ઓનલાઈન Github એડીટર નો ઉપય઼ોગ કરો)।
+- નીચે દશાૅવેલ કમાંડ થી એક નવી બ્રાંચ બનાવો.
+```
+git checkout -b નવી-બ્રાંચ-નુ-નામ
+```
+- જે ફાઇલ તમે ટ્રાંસલેટ કરવા ઈચ્છો છો તે ફાઇલ માં જાવ અને સંપુણૅ ફાઇલ ને ગુજરાતી માં ટ્રાંસલેટ કરો.
+- ટ્રાંસલેશન પુર્ણ કર્યા બાદ, બધા સુધારા ને નીચે આપેલા કમાંડ થી `add`, `commit` અને `push` કરો.
+```
+git add ફાઇલ-નુ-નામ
+git commit -m "તમારા સુધારા વિશે લખો"
+git push origin નવી-બ્રાંચ-નુ-નામ
+```
+
+# રિવ્યુ પ્રોસેસ
+
+બધી પૂલ રિક્વેસ્ટ ને ૨ રિવ્યુઅર પાસે થી માન્ય થયેલ હોવી જરુરી છે. રીવ્યુ માટે નીચે ની બાબતો નુ ધ્યાન થી પાલન કરો.
+
+- બધા શબ્દો નુ પૂલ રીક્વેસ્ટ માં અંગ્રેજી વર્ણન કરો. જેમને રીવ્યુઅર રીવ્યુ પુર્ણ થયા બાદ ચેક માર્ક ક્રશે.
+- કોઇ પણ રીવ્યુઅર ના રીવ્યુ ફીડબૈક ને રીસોલ્વ માર્ક ના કરવુ. રીવ્યુઅર તમારા ફીડબૈક સુધારા તપાસ્યા બાદ રીસોલ્વ માર્ક કરશે.
+- તમે ફીડબૈક ના સુધારા કર્યા પછી તે ફીડબૈક પર કમેંટ કરો અથવા 👍 (ઇમોજી) લગાવી આપો. જેનાથી રીવ્યુઅર ને ખ્યાલ આવી શકે કે તમે આ ફીડબૈક ને સુધારી નાખ્યા છે.
+> પૂલ રીક્વેસ્ટ ને મર્જ થવા માટે ઓછા માં ઓછા એક રીવ્યુઅર ની અપ્રુવલ હોવી જરુરી છે.
```

Comment:
@kishanjasani this line is contradicted with line no 26, which says contributor requires 2 approved reviews for pull request merge.

Reply:
Yes, I will have to remove that.